include/ThingProperties.h

```diff
@@ -28,6 +28,12 @@ void updateBatteryVoltage(float voltage) {
     battery_Voltage = voltage;
 }
 
+void updateAcceleration(accel_t acceleration_struct){ 
+    new_accel_x = acceleration_struct.accel_x; 
```

Comment:
I don’t see new_accel_ variables in this file. They should be with the battery_voltage variable.